packages/core/src/codewhisperer/util/zipUtil.ts

```diff
@@ -8,8 +8,9 @@ import path from 'path'
 import { tempDirPath } from '../../shared/filesystemUtilities'
 import { getLogger } from '../../shared/logger'
 import * as CodeWhispererConstants from '../models/constants'
+import { ToolkitError } from '../../shared/errors'
 import { fsCommon } from '../../srcShared/fs'
-import { statSync } from 'fs'
+import { collectFiles } from '../../amazonqFeatureDev/util/files'
```

Comment:
Nice! Glad this was able to be re-used. That really gives us leverage.

In a separate PR, consider moving `collectFiles` somewhere out of "featureDev" (perhaps to the existing `workspaceUtils.ts` module)

Reply:
Thanks! I will work on that separately